### PR TITLE
Gather fluentd logs in e2e runs

### PR DESCRIPTION
**Pull request by JAORMX**
This adds a small script that gathers the fluentd logs from the nodes.

### CODE REVIEW

File: hack/lib/util/logs.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+get_logging_pod_logs() {
+  local node
+  for node in $(oc get nodes --no-headers | awk '{print $1}') ; do
+      oc adm node-logs $node --path=fluentd/fluentd.log > $ARTIFACT_DIR/$node-fluentd.log
```
Comment (richm) [inline hack/lib/util/logs.sh]: hmm - is there some way to use `oc adm node-logs` to grab `fluentd/fluent.*.log` too?  If there is a lot of output, fluentd will do log rotation, and you will want to grab all of them, rotated logs too.

File: hack/lib/util/logs.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+get_logging_pod_logs() {
+  local node
+  for node in $(oc get nodes --no-headers | awk '{print $1}') ; do
```
Comment (richm) [inline hack/lib/util/logs.sh]: the usual idiom for getting only the name of some object is to use `-o jsonpath='{.items[*].metadata.name}'` - so
```bash
for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}')
```

File: hack/lib/util/logs.sh (hunk 1 above)
Comment (JAORMX) [reply hack/lib/util/logs.sh]: Let me give it a try; I think it's possible

File: hack/lib/util/logs.sh (hunk 2 above)
Comment (JAORMX) [reply hack/lib/util/logs.sh]: thanks!

**Pull request merged**